### ISSUE TITLE
Any word on emblem rails working with ember-1.10.0 

### ISSUE THREAD

**Issue by danielspaniel** [alexspeller/emblem-rails#36]
Wondering if anyone has got an ember rails application running with ember-1.10.0 ??

 I have tried just about every combination of things to get this working, but no luck.
 But if anyone has had success that I would be really thrilled to hear how they did it.

**Comment by machty** [comment]
Work on this should be finished within the week with the next version of Emblem; progress can be tracked here: https://github.com/bantic/emblem.js/tree/ember-cli

**Comment by danielspaniel** [comment]
Was a supremely good idea to just compile to handlebars syntax by the way .. and thanks for the update. Look forward to the new release.

**Comment by danielspaniel** [comment]
I noticed that the new version of emblem 0.5.0 is now out, and there is a special add on for ember-cli:
https://github.com/201-created/ember-cli-emblem-hbs-printer 
 Is there something like this for ember-rails?  or is emblem ready to go now with the current versions of barber-emblem and emblem-rails?

**Comment by jimmay5469** [comment]
@danielspaniel I think this needs resolved first: https://github.com/machty/emblem.js/issues/216

**Comment by coladarci** [comment]
Any update here? Still can't get anything to compile and there are so many moving pieces, I don't know what to track.. Any guidance would be appreciated!

**Comment by alexspeller** [comment]
I'm unlikely to work on this, I don't use ember-rails anymore. PRs welcome, but I strongly reccomend moving to ember-cli as soon as possible.

**Comment by coladarci** [comment]
Thanks for the quick reply - completely agree with the need to switch over - massive code base so unfortunately not a trivial task on our part.. It sounds like the majority of the work needs to be done else where, so maybe those folks are still more into fixing this.

**Comment by danielspaniel** [comment]
Yeah .. I feel for you on having to push the whole codebase over the ember-cli mountain, and for being let down by @matchy.  I just spent 2 weeks moving to cli because I see that emblem is finished in sprockets land.   They got emblem working for cli and stopped there.  So, you either convert all your templates to html/hbs ( and stay on ember rails )  or convert the js code to es6 and go to cli.  I found a gem 'ember-cli-rails' that makes the conversion alittle easier at least.  It allows your rails code to hook into the cli process. 
 


     On Sunday, March 29, 2015 12:42 PM, coladarci <notifications@github.com> wrote:
   

 Thanks for the quick reply - completely agree with the need to switch over - massive code base so unfortunately not a trivial task on our part.. It sounds like the majority of the work needs to be done else where, so maybe those folks are still more into fixing this.—

**Comment by coladarci** [comment]
Can't say I'm let down by @machty - the work he and everyone else has done / is doing has made our code base amazing. There are so many exciting things happen in CLI, I can't wait to get it in our stack, just going to be a tad painful.

**Comment by cheerfulstoic** [comment]
Am I right in seeing that ember-cli requires a dependency on npm?  I have a gem that I wrote using emblem-rails and I don't want to have everybody that uses my gem have to install npm...

**Comment by cheerfulstoic** [comment]
(to clarify a bit, my gem is a Rails engine)

**Comment by alexspeller** [comment]
@cheerfulstoic I suggest compiling your code with ember-cli and then publishing the gem with the output of that code

**Comment by cheerfulstoic** [comment]
Got it, I'll give that a try.  Thanks!

**Comment by botandrose** [comment]
Hey folks, I just released the extremely poorly named `ember-emblem-template` gem: https://github.com/botandrose/ember-emblem-template. This should be a replacement for emblem-rails, if you want to use Emblem 0.5 on Ember 1.10+ with Rails.

@alexspeller What do you think about something like this becoming emblem-rails 0.3.0?

**Comment by botandrose** [comment]
Note that it doesn't require Ember 1.10+, just facilitates upgrading to it.

**Comment by Eric-Guo** [comment]
@botandrose thanks, it works perfect to below gems combination:

```ruby
gem 'ember-rails', '~> 0.19.0'
gem 'ember-source', '~> 1.11.3.1'
gem 'ember-emblem-template'
gem 'emblem-source', '~> 0.4'
```

**Comment by Jsericks** [comment]
@botandrose -- Thanks! Thought I was losing my mind trying to figure this one out.
